File: app/src/main/java/com/pandatone/kumiwake/kumiwake/KumiwakeSelectMode.java
package com.pandatone.kumiwake.kumiwake;

import android.content.Intent;
import android.os.Bundle;
import android.support.v4.app.NavUtils;
import android.support.v4.content.ContextCompat;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.MenuItem;
import android.widget.LinearLayout;
import android.widget.TextView;

import com.pandatone.kumiwake.R;
import com.pandatone.kumiwake.sekigime.SekigimeResult;

import butterknife.ButterKnife;
import butterknife.OnClick;

/**
 * Created by atsushi_2 on 2016/02/19.
 */
public class KumiwakeSelectMode extends AppCompatActivity {
    public static boolean sekigime = false;
    TextView normalmode,quickmode;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.kumiwake_select_mode);

        Toolbar toolbar = (Toolbar) findViewById(R.id.tool_bar_1);
        setSupportActionBar(toolbar);

        getSupportActionBar().setTitle(R.string.kumiwake);
        getSupportActionBar().setDisplayShowTitleEnabled(true);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        normalmode=(TextView)findViewById(R.id.descriptionForNormalMode);
        quickmode=(TextView)findViewById(R.id.descriptionForQuickMode);
        if(sekigime){
            toolbar.setBackgroundColor(getResources().getColor(R.color.green_title));
            LinearLayout layout=(LinearLayout) findViewById(R.id.select_mode_layout);
            layout.setBackgroundColor(getResources().getColor(R.color.green_background));
            normalmode.setText(R.string.description_of_normal_sekigime);
            quickmode.setText(R.string.description_of_quick_sekigime);
            getSupportActionBar().setTitle(R.string.select_mode);
            getSupportActionBar().setDisplayHomeAsUpEnabled(false);
        }
        ButterKnife.bind(this);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        int id = item.getItemId();

        // いつものUPナビゲーションの処理
        switch (id) {
            case android.R.id.home:
                NavUtils.navigateUpFromSameTask(this);
                return true;
        }
        return super.onOptionsItemSelected(item);
    }

    @OnClick(R.id.normal_mode_button) void onNormalModeClicked() {
        SekigimeResult.Normalmode = true;
        startActivity(new Intent(this, NormalMode.class));
    }

    @OnClick(R.id.quick_mode_button) void onQuickModeClicked() {
        SekigimeResult.Normalmode = false;
        startActivity(new Intent(this, QuickMode.class));
    }
}

File: app/src/main/java/com/pandatone/kumiwake/kumiwake/KumiwakeSelectMode.kt
package com.pandatone.kumiwake.kumiwake

import android.content.Intent
import android.graphics.drawable.AnimationDrawable
import android.os.Bundle
import android.view.MenuItem
import android.view.View
import android.widget.LinearLayout
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import androidx.core.app.NavUtils
import androidx.core.content.ContextCompat
import butterknife.ButterKnife
import butterknife.OnClick
import com.pandatone.kumiwake.R
import com.pandatone.kumiwake.sekigime.SekigimeResult
import kotlinx.android.synthetic.main.add_member.*
import kotlinx.android.synthetic.main.kumiwake_select_mode.*

/**
 * Created by atsushi_2 on 2016/02/19.
 */
class KumiwakeSelectMode : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.kumiwake_select_mode)

        val toolbar = findViewById<View>(R.id.tool_bar_1) as Toolbar
        setSupportActionBar(toolbar)

        supportActionBar!!.setTitle(R.string.kumiwake)
        supportActionBar!!.setDisplayShowTitleEnabled(true)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)

        val layout = findViewById<View>(R.id.select_mode_layout) as LinearLayout

        if (sekigime) {
            toolbar.setBackgroundColor(ContextCompat.getColor(applicationContext, R.color.green_title))
            layout.background = getDrawable(R.drawable.gradient_anim_sekigime)
            descriptionForNormalMode.setText(R.string.description_of_normal_sekigime)
            descriptionForQuickMode.setText(R.string.description_of_quick_sekigime)
            supportActionBar!!.setTitle(R.string.select_mode)
        }else{
            layout.background = getDrawable(R.drawable.gradient_anim_kumiwake)
        }
        val animDrawable = layout.background as AnimationDrawable
        animDrawable.setEnterFadeDuration(resources.getInteger(R.integer.anim_duration))
        animDrawable.setExitFadeDuration(resources.getInteger(R.integer.anim_duration))
        animDrawable.start()

        ButterKnife.bind(this)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {

        // いつものUPナビゲーションの処理
        when (item.itemId) {
            android.R.id.home -> {
                NavUtils.navigateUpFromSameTask(this)
                return true
            }
        }
        return super.onOptionsItemSelected(item)
    }

    @OnClick(R.id.normal_mode_button)
    internal fun onNormalModeClicked() {
        SekigimeResult.Normalmode = true
        startActivity(Intent(this, NormalMode::class.java))
    }

    @OnClick(R.id.quick_mode_button)
    internal fun onQuickModeClicked() {
        SekigimeResult.Normalmode = false
        startActivity(Intent(this, QuickMode::class.java))
    }

    companion object {
        var sekigime = false
    }
}

